docs/api-reference/core/transform.md

```diff
@@ -118,7 +118,7 @@ Constructs a `Transform` object. It then creates destination buffers if needed a
   * `sourceBuffers` (`Object`) - key and value pairs, where key is the name of vertex shader attribute and value is the corresponding `Attribute`, `Buffer` or attribute descriptor object.
   * `feedbackBuffers` (`Object`, Optional) - key and value pairs, where key is the name of vertex shader attribute and value is the corresponding `Buffer` object.
   * `vs` (`String`) - vertex shader string.
-  * `varyings` (`Array`) - Array of vertex shader varyings names.
+  * `varyings` (`Array`) - Array of vertex shader varyings names. When not provided this can be deduced from `feedbackMap`. Either `varyings` or `feedbackMap` must be provided.
```

Comment:
In luma.gl v6 varyings are deduced automatically from the Model's program. Do we still need this?

Reply:
Not all varyings defined in vertex shader have to be captured, ie. this varyings array could be subset of all varyings.